app/sanitizers/feed_markdown_scrubber.rb

```diff
@@ -8,6 +8,6 @@ def initialize
   def allowed_node?(node)
     return true if tags.include?(node.name) && node.name != "a"
 
-    node.name == "a" && !node["href"].start_with?("#")
+    node.name == "a" && !node["href"]&.start_with?("#")
```

Comment:
my brain had to stare and rethink this for a bit
- if the node is an anchor element `<a>`
- if there is no href, `!node['href']` is true
- if there is an absolute link, `!node['href'].start_with?('#')` is true
- if the link is page-relative, `<a href='#'>`, then `!node[href].start_with?('#')` is false, scrub this tag.